docs/cli.md

```diff
@@ -310,6 +311,11 @@ You can do this using the `add` command.
 When `--only` is specified, `--with` and `--without` options are ignored.
 {{% /note %}}
 
+{{% note %}}
+`poetry update` without any options will only update the main dependencies of the project.
```

Comment:
I think this is pedantically not correct - non-optional dependency groups are also updated, right?

Reply:
That is probably right than non-optional dependency groups are also updated, although I have not tested this; updated, thanks.